### PR TITLE
Add test for SELinux: restorecon + chcon + chcat

### PR DESCRIPTION
**Pull request by lilyeyes**
Add test for SELinux: restorecon + chcon + chcat
All passed.

- Related ticket:
  https://progress.opensuse.org/issues/65998
  https://progress.opensuse.org/issues/66093
  https://progress.opensuse.org/issues/66096
- Needles: NA
- Verification run: 
  https://openqa.nue.suse.com/tests/4169718# (old)
  https://openqa.nue.suse.com/tests/4181769# (new)

### CODE REVIEW

File: lib/selinuxtest.pm [hunk 1]
```diff
@@ -73,4 +73,16 @@ sub check_fcontext {
     }
 }
 
+# check SELinux security category of a file/dir
+sub check_category {
```
Comment (jouyingbin) [inline lib/selinuxtest.pm:77]: May I know which case is using the **check_category** function, actually I did not find the case actually. Thanks.

File: tests/security/selinux/chcat.pm [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    validate_script_output(
+        "chcat -L",
+        sub {
+            m/
+	    $systemlow\ .*SystemLow.*
+            $systemlow_systemhigh\ .*SystemLow-SystemHigh.*
+            $systemhigh\ .*SystemHigh/sx
+        });
+
+    # test "# chcat [+|-]CATEGORY File" can operate categories on dirs/files
```
Comment (jouyingbin) [inline tests/security/selinux/chcat.pm:58]: It looks like you use **check_fcontext** to check the category, am I right?

File: lib/selinuxtest.pm (hunk 1 above)
Comment (lilyeyes) [reply lib/selinuxtest.pm:77]: good catching, I used/pasted wrong function (copy and paste do harms sometime).

File: tests/security/selinux/chcat.pm (hunk 2 above)
Comment (lilyeyes) [reply tests/security/selinux/chcat.pm:58]: Yes, "chcat" should invoke "", but "chcat" passed, weired :(.
I have checked the log, the "check_fcontext " did not take effect. so test case "chcat" passed.
I will revise the code, thanks :)

File: lib/selinuxtest.pm (hunk 1 above)
Comment (lilyeyes) [reply lib/selinuxtest.pm:81]: Hi Ben,
I have revised the code and did rerun (see new test run link above).
All cases are pass.
Please help to check again.

File: tests/security/selinux/chcat.pm [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    # create a testing directory/file
+    $self->create_test_file("$test_dir", "$test_file");
+
+    # test "# chcat -L" can list available categories
+    validate_script_output(
+        "chcat -L",
+        sub {
+            m/
+	    $systemlow\ .*SystemLow.*
```
Comment (jouyingbin) [inline tests/security/selinux/chcat.pm:53]: This line seems not to align with others, does this change after CI check automatically?

File: lib/selinuxtest.pm (hunk 1 above)
Comment (jouyingbin) [reply lib/selinuxtest.pm:81]: Sure

File: tests/security/selinux/chcat.pm (hunk 3 above)
Comment (lilyeyes) [reply tests/security/selinux/chcat.pm:53]: It aligns with others: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/b69f848a43a217987c7fffede0131a5be4c55206/tests/security/selinux/chcat.pm
But not sure why it looks like unaligned.

Comment (jouyingbin) [reply tests/security/selinux/chcat.pm:53]: okay, perhaps the display problem, not a big deal. :)

**Pull request merged**